examples/authentication/headless/src/pages/example/index.tsx

```diff
@@ -55,6 +55,7 @@ export const ExamplePage: React.FC = () => {
             label: "Sign in with GitHub",
         },
     ];
+    // eslint-disable-next-line
     const Box = ({ children }: any) => (
```

Comment:
```suggestion
    const Box = ({ children }: React.PropsWithChildren) => (
```

Will be better to type this rather than suppressing the warning 😅